src/core/common/Topbar/fragments/LowPart.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import { withStyles } from '@material-ui/core/styles';
+import Toolbar from '@material-ui/core/Toolbar';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import Menu from '@material-ui/core/Menu';
+
+import PageTitle from "./PageTitle";
+import styles from "../../../styles";
+
+/**
+ * This component returns green (bottom) part of Showcase TopBar
+ *
+ * @author Bogdan Shcherban <bsc@piogroup.net>
+ * @constructor
+ */
+const GreenPart = ({ classes, setSidebarVisibility, isSidebarOpen, isMenuVisible, location, patientInfo }) => {
```

Comment:
I try to name my components according to their function rather than their aspect, because Aspect can change. "GreenPart" doesn't really make sense here as it's not green?

Reply:
It has been renamed